deltachat-ffi/deltachat.h

```diff
@@ -463,6 +463,64 @@ char*           dc_get_info                  (const dc_context_t* context);
 char*           dc_get_oauth2_url            (dc_context_t* context, const char* addr, const char* redirect_uri);
 
 
+#define DC_CONNECTIVITY_NOT_CONNECTED        1000
+#define DC_CONNECTIVITY_CONNECTING           2000
+#define DC_CONNECTIVITY_WORKING              3000
+#define DC_CONNECTIVITY_INTERRUPTING_IDLE    4000
```

Comment:
I don't like the name of this state, "interrupting idle" is an implementation detail. The fact that we have an "interrupt idle" operation inside the scheduler should not be leaked to the API. Maybe rename it to `CHECKING` or something like this. And `WORKING` to `FETCHING` to be more precise.

Am I right that after `dc_maybe_network` you get to `INTERRUPTING_IDLE` first, and then either to `WORKING` and then `CONNECTED` (if there are messages) or immediately to `CONNECTED` (if there are no messages)?

Reply:
> Am I right that after `dc_maybe_network` you get to `INTERRUPTING_IDLE` first, and then either to `WORKING` and then `CONNECTED` (if there are messages) or immediately to `CONNECTED` (if there are no messages)?

Yes. And that's the only way to get `INTERRUPTING_IDLE`, if IDLE was interrupted by timeout or a new message from the server, you don't get `INTERRUPTING_IDLE`.

> Maybe rename it to `CHECKING` or something like this.

Yet another idea: Just wondering if we should call it `MAYBENETWORK_CALLED`.

BTW with @r10s we also wondered if we should rename `CONNECTED` to `IDLE`. In this case, it might make sense to use `IDLE_MAYBENETWORK_CALLED` instead of just `MAYBENETWORK_CALLED` to indicate that those are actually the same connectivity (we only distinguish them so that after calling `dc_maybe_network()` it is enough to wait for `CONNECTED` and you can be sure that things were fetched once).

Reply:
What do you think about all the other possible renamings I mentioned?